api/Element.json

```diff
@@ -5929,6 +5929,41 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "escape_lt_gt_in_attributes": {
+          "__compat": {
+            "description": "Serializes `<` and `>` in attributes as `&lt;` and `&gt;`.",
+            "spec_url": "https://github.com/whatwg/html/issues/6235",
```

Comment:
This is the URL of the issue driving this. No spec yet. Can I use this? If so can you please remind me how.

Reply:
See: https://github.com/mdn/browser-compat-data/pull/26688#discussion_r2075540299